src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,40 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  let columnTitle: Array<object> = Object.keys(
+    languages
+  ).map((title: string) => ({ type: "text", title: `${title}`, width: 150 }));
+  return columnTitle;
+};
+
+const setKeys = (languages: LanguagesInput): Array<string> => {
+  const keyValue = new Set();
+  const reducer = (accumulator: object, currentValue: object) =>
+    keyValue.add(Object.keys(currentValue));
+  Object.keys(languages)
+    .map((titles) => languages[titles])
+    .reduce(reducer);
+  return keyValue.values().next().value;
```

Comment:
are you planning to write only first value or all values?

Reply:
Only first value